modules/pattern/pattern.service.js

```diff
@@ -137,14 +137,15 @@ class PatternsService {
       }).exec();
     }
 
-    const uploadResult = await uploadService.uploadFile(image[0]);
-    const images = uploadResult.fileNames;
-    const constructorImg = await uploadSmallImage(image[1]);
-    pattern.constructorImg = constructorImg;
-
-    if (!images && constructorImg) {
-      return Pattern.findByIdAndUpdate(id, pattern).exec();
+    if (image[0].file) {
```

Comment:
For my opinion it's bad naming... (image[1] && image[0].file) They are different type objects? Can u call them somehow?

Reply:
changed to destructuring